engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowExecutionActor.scala

```diff
@@ -64,6 +80,13 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId, workflowDescript
   }
 
   when(WorkflowExecutionInProgressState) {
+    case Event(BackendJobExecutionSucceededResponse(jobKey, callOutputs), stateData) =>
+      log.info(s"Job ${jobKey.call.fullyQualifiedName} succeeded! Outputs: ${callOutputs.mkString(" |:::| ")}")
```

Comment:
That seems like it'd be more readable. Although I'm questioning if the outputs really need to be in the log in the first place, I suppose we could always remove it later if it ends up being too chatty